Source/EventFlow/Configuration/IEventFlowConfiguration.cs

```diff
@@ -86,5 +87,17 @@ public interface IEventFlowConfiguration
         CancellationBoundary CancellationBoundary { get; }
 
         bool ForwardOptimisticConcurrencyExceptions { get; set; }
+        
+        /// <summary>
+        /// Use <c>EventNamingStrategy</c> to determine the internal naming strategy
+        /// for events. The strategy is applied whenever an event is persisted
+        /// and loading from persistence and when serializing/deserializing in that
+        /// context respectively.
+        /// 
+        /// If more fine grained control of is needed, a custom implementation of
+        /// <c>IEventNamingStrategy</c> should be provided.
+        /// </summary>
+        /// <remarks>Defaults to SimpleNamingStrategy</remarks>
+        IEventNamingStrategy EventNamingStrategy { get; }
```

Comment:
```suggestion
```

Don't think this is used anywhere, might be leftovers from the original commit

Reply:
Right, simply leftovers. I had them actually on my "to do before push" list